### PR TITLE
use higher default FEE_PER_KB and FEE_SECURITY_MARGIN

### PR DESCRIPTION
**Pull request by gabegattis**

### CODE REVIEW

**Comment by gabegattis** [comment]
I increased each value by a factor of 10. This is arbitrary, but I don't know what else to set them to. The transaction fee market is extremely volatile. If the recent market trends continue, we could be back in this same situation soon. Users of bitcore-lib really need to use an outside source to get a suggested fee/kb.

I am starting to question the wisdom of having these values at all. I am wondering if we should remove them in the next major release.

Feedback is appreciated.

**Comment by matiu** [comment]
LGTM.

 The original value was also arbitrary. It is just a very high value in case the programmer make and error and the fee turns out very high.

**Comment by visvirial** [comment]
Hi, I am the original reporter of this issue: #123 

There are some options to fix this problem.
1. Increase the default value as is done in this PR.
1. Remove the checking functionality to allow arbitrary high fees.
1. Fetch recommended fee rate from some web services, such as https://bitcoinfees.21.co/.
1. Allow users to override the default value easily (by passing some options?).
1. (Other options. Reply as a comment)

There are some cons. and pros. for each options. How do you guys feel?

**Comment by gabegattis** [comment]
@visvirial We have already talked about removing the checking functionality entirely in the next major (api breaking) release. We have not made a decision yet, but most of us think that it would be a good idea.

bitcore-lib is meant to be a completely offline library, so we will not add a feature to fetch from external sources. We already make this data available at https://github.com/bitpay/insight-api#utility-methods Users of bitcore-lib should get fee recommendations form a source like that.

Allowing users to override the default constants is another option. We might do this if we decide to keep the checking functionality long term.

**Comment by gabegattis** [comment]
@matiu @pnagurny @kleetus I fixed the tests. This is ready for review now.

File: test/data/tx_creation.json [hunk 1]
```diff
@@ -80,6 +80,6 @@
...
...524a6f956a229e69d31aef6c7a98b2a291f3cfc6701483045022100e6ae6c43240e8a11a6de2d034501c2a366c0ccdf069c7828de0791f05e68e787022028b80bd36c2b2ae63fe7afb491da6c0ce23fbbb982450962c817b20f0bb24075014c695221020483ebb834d91d494a3b649cf0e8f5c9c4fcec5f194ab94341cc99bb440007f2210271ebaeef1c2bf0c1a4772d1391eab03e4d96a6e9b48551ab4e4b0d2983eb452b2103a659828aabe443e2dedabb1db5a22335c5ace5b5b7126998a288d63c99516dd853aeffffffffa0644cd1606e081c59eb65fe69d4a83a3a822da423bc392c91712fb77a192edc00000000fc00483045022100ae7f136cf906dc37d34d5035b8d2001c6a783773b74507ba83080e73e903623f0220023baf7738395268f7097e5586130f682b911fd49b83b265f8fa481f2a6b1ee90146304302201d60f512a8b37663d85c123933053e0354f13d89daf699ca600defa03d4a1dab021f41042b6e4ba30311fc3a68c228c3725f3b0f05a4453ef19408e6a4ae30a2b0014c695221020483ebb834d91d494a3b649cf0e8f5c9c4fcec5f194ab94341cc99bb440007f2210271ebaeef1c2bf0c1a4772d1391eab03e4d96a6e9b48551ab4e4b0d2983eb452b2103a659828aabe443e2dedabb1db5a22335c5ace5b5b7126998a288d63c99516dd853aeffffffff03f04902000000000017a9144de752833233fe69a20064f29b2ca0f6399c8af387007102000000000017a9144de752833233fe69a20064f29b2ca0f6399c8af387ab2f03000000000017a9146c8d8b04c6a1e664b1ec20ec932760760c97688e8700000000"
```
Comment (gabegattis) [inline test/data/tx_creation.json]: When reviewing, parse these txs and do a toObject() on them. You will see that the old one used a 10000 satoshi fee. The new one is 10x higher. Everything else is the same

Comment (kleetus) [inline test/data/tx_creation.json]: LGTM

**Comment by kleetus** [comment]
Review complete. LGTM.

**Pull request merged**